README.md

```diff
@@ -14,8 +14,20 @@ functionality, you can use this bookmarklet to temporarily disable all
 WYSIWYG A/B tests, which should restore the old editor functionality.
 
 ## Usage
+
+### In the browser
+
 1. Copy the contents of `index.js`
 2. Paste into the URL field of a browser bookmark
 3. Navigate to a Slack workspace in that browser
 4. Execute the bookmarklet to disable the WYSIWYG editor
 5. Repeat each time you reload slack
+
+### In the desktop app
+
+1. Open the slack app in Developer mode
+  * On Mac: `export SLACK_DEVELOPER_MENU=true && open -a /Applications/Slack.app`
```

Comment:
```suggestion
  * On Mac: `export SLACK_DEVELOPER_MENU=true && open -a /Applications/Slack.app`
  * On other OS's: ensure the `SLACK_DEVELOPER_MENU` environment variable is set to `true` before opening the Slack app.
```

Reply:
Thanks for the amendments! Doesn't look like I have permissions to push to this branch, so I'm going to merge as-is, and we can follow up with these changes after.